torch_xla/csrc/ir.h

```diff
@@ -138,6 +138,12 @@ class XlaNode : public torch::lazy::Node {
 
   std::string ToString() const override;
 
+  void MarkDynamicDimension(uint32_t dim) { dynamic_dims_.push_back(dim); }
```

Comment:
what if same dim is being called multiple times? Shouldn';t it be a `unordered_set` or boolean array?

Reply:
Good point, let me change to `unordered_set`